ivy/functional/frontends/numpy/random/functions.py

```diff
@@ -97,6 +97,14 @@ def standard_gamma(shape, size=None):
     return ivy.gamma(shape, 1.0, shape=size, dtype="float64")
 
 
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def standard_t(df, size=None):
+    numerator = ivy.random_normal(mean=0.0, std=1.0, shape=size, dtype="float64")
+    denominator = ivy.gamma(df / 2, 1.0, shape=size, dtype="float64")
+    return ivy.sqrt(df / 2) * ivy.divide(numerator, ivy.sqrt(denominator))
```

Comment:
Should I be using the division function implemented in ivy here (`ivy.divide`)? I tried to do that but the gamma function fails because it expects to have `integers` or `floats` as inputs rather than `ivy.arrays`.

Reply:
Gamma accepts both `Array` and `NativeArray` so this shouldn't be an issue

Reply:
```suggestion
    denominator = ivy.gamma(ivy.divide(df, 2), 1.0, shape=size, dtype="float64")
    return ivy.sqrt(ivy.divide(df, 2)) * ivy.divide(numerator, ivy.sqrt(denominator))
```

When I run the tests locally with `ivy.divide` I get an error saying:

```error
ivy.utils.exceptions.IvyBackendException: numpy: gamma: low and high bounds must be numerics when shape is specified
```

Reply:
Error on the CI seems to be somewhat different anbd suggest input issues. Please check the same https://github.com/unifyai/ivy/actions/runs/5811559595/job/15755368193?pr=21549#step:3:543